lib/src/over_react_test/jacket.dart

```diff
@@ -203,7 +202,7 @@ class TestJacket<T extends react.Component> {
           'getDartInstance() is only supported when the rendered object is a composite (class based) component.');
     }
 
-    return over_react.getDartComponent(_renderedInstance) as T;
+    return over_react.getDartComponent(_renderedInstance) as T?;
```

Comment:
If we update the `T` constraint like mentioned above, then this explicit cast could be removed:
```suggestion
    return over_react.getDartComponent(_renderedInstance);
```